toolchain/Config.in

```diff
@@ -238,6 +238,7 @@ comment "C Library"
 choice
 	prompt "C Library implementation" if TOOLCHAINOPTS
 	default LIBC_USE_UCLIBC if arc
+	default LIBC_USE_GLIBC if powerpc64
 	default LIBC_USE_MUSL
```

Comment:
ok, i just started a test build on my server. if it goes well, i'll queue this for post the release